.github/test-spec.yml

```diff
@@ -31,7 +31,17 @@
   - "samples/subsys/mgmt/mcumgr/smp_svr/**/*"
 
 "CI-tfm-test":
-  - "**/*"
+  - "boards/arm/nrf5340dk_nrf5340/**/*"
+  - "boards/arm/nrf9160dk_nrf9160/**/*"
+  - "drivers/entropy/**/*"
```

Comment:
Because of PSA entropy driver?

Reply:
Yes, I can reduce the filter to only check that file